e2e/terraform/config/full-cluster/consul/server/server.json

```diff
@@ -0,0 +1,8 @@
+{
+  "server": true,
+  "ui": true,
+  "bootstrap_expect": 3,
+  "service": {
+    "name": "consul"
+  }
+}
```

Comment:
Note to reviewers: this is a full copy of the moved config now at `./e2e/terraform/config/dev-cluster/consul/server/server.json`